docs/user_guide.md

```diff
@@ -4,63 +4,146 @@ This is a pre-release version of the extension and is not intended for general u
-mechanisms as needed on a query-by-query basis to minimize noise while ensuring strong anonymity.
+The [Open Diffix website](https://www.open-diffix.org) has general information about the Open Diffix project. The [website FAQ](https://www.open-diffix.org/faq) is a good starting point. [This article](https://www.open-diffix.org/blog/diffix-elm-automates-what-statistics-offices-have-been-doing-for-decades) is a high-level overview of the anonymization mechanisms used by Diffix. [This paper](https://arxiv.org/abs/2201.04351) is a complete specification and privacy analysis of Diffix (version Elm).
 
-## What is Diffix for PostgreSQL?
+## Configuration objects
 
-__Diffix for PostgreSQL__ is an implementation of the Diffix mechanism for PostgreSQL databases.
-It is provided as an extension module for PostgreSQL version 13 or higher.
+Extension behavior is controlled by `security labels` and `settings`. Security labels specific to this extension are assigned to tables, columns, and roles (users). Settings are general parameters associated with system operation. Settings can be assigned in the configuration file and are instantiated per session.
 
-## Configuring the extension
+Only superusers can assign security labels and settings. To remove a security label from an object, set it to `NULL`.
 
-System behaviour can be configured using a combination of custom variables and security labels.
+The command `SELECT * FROM diffix.show_labels();` displays the current security labels assigned to tables and columns by the extension.
 
-### Labeling database objects
+The command `SELECT * FROM diffix.access_level();` displays the current security labels assigned to the role associated with the active session.
 
-The module acts as a security provider and allows the marking of database objects with various anonymization labels,
-which associate additional metadata, needed for anonymization, with existing objects. For more details about security
-labels, see the official [documentation page](https://www.postgresql.org/docs/current/sql-security-label.html).
+The command `SELECT * FROM diffix.show_settings();` displays the settings in use by the extension for the active session.
 
-Only superusers can set anonymization labels.
-To remove an anonymization label from an object, set it to `NULL`.
+NOTE: If no configuration is done, then by default no anonymization takes place. All users will have full access to table contents.
 
-Execute `SELECT * FROM diffix.show_labels();` to display the current labels in use by the extension.
+For more information about PostgreSQL security labels, see the official [documentation page](https://www.postgresql.org/docs/current/sql-security-label.html).
 
-Tables can be labeled as `public` or `personal`. Direct access is allowed to public data even for restricted users.
-Unlabeled tables can not be queried (unless `treat_unmarked_tables_as_public` is set to `true`).
+## Security labels
```

Comment:
not wanting to suggest a total overhaul, I start to see that the narrative here is (and was) too much centered around security labels, which are more of a means to an end, and a very specific postgreSQL thing.

Could we perhaps turn it around and clarify setting roles/tables/columns up without the "security label" cruft, and only at the end or beginning mention that those are all PostgreSQL security labels and one can read more in <link to postgres docs>.

This would put the main subject (configuring anonymization) first.